crates/bevy_light/src/cascade.rs

```diff
@@ -225,8 +225,8 @@ pub fn build_directional_light_cascades(
         // users to not change any other aspects of the transform - there's no guarantee
         // `transform.to_matrix()` will give us a matrix with our desired properties.
         // Instead, we directly create a good matrix from just the rotation.
-        let world_from_light = Mat4::from_quat(transform.compute_transform().rotation);
-        let light_to_world_inverse = world_from_light.inverse();
+        let world_from_light = Mat4::from_quat(transform.rotation());
+        let light_to_world_inverse = world_from_light.transpose();
```

Comment:
Should this now be `light_to_world_transpose`? Do you think a comment should be here explaining why this is still valid?

Reply:
it should be light_from_world tbh

Reply:
theres a few other inconsistencies in the file, i'd rather leave them to a follow up so as to not merge conflict myself. variable renames are trivial, im mostly interested in getting the precision improvements in atm